framework/bug-mining/README.md

```diff
@@ -78,7 +78,7 @@ the `commit-db` for Chart):
 | Project ID | Issue tracker | Issue tracker project ID | Regexp                    |
 |------------|---------------|--------------------------|---------------------------|
 | Chart      |               |                          |                           |
-| Closure    | google        | closure-compiler         | `/issue.*(\d+)/mi`        |
+| Closure    | google        | closure-compiler         | `/issue.*[^\d](\d+)/mi`   |
```

Comment:
Actually, the old regex was only matching the last digit of issues ids. So, it was matching the same number of issues but the matched issue id was incomplete.

Reply:
There is a subtle difference:
* The old regex made the characters before the captured issue ID optional. The issue to solve is: the matching is greedy, and the `.*` consumes most of the issue ID.
* The new regex requires a single non-digit character before the issue ID.

In the new regex, should the non-digit character also be optional and possibly capture more than one (`[^\d]*` vs. `[^\d]?`)?

Reply:
> In the new regex, should the non-digit character also be optional and possibly capture more than one (`[^\d]*` vs. `[^\d]?`)?

Because we might found a commit message like `Issue123`? I've just checked closure gitlog and I didn't find any case like that...  But, we might find examples like in the future.

I've just tried `/issue.*[^\d](\d+)/mi`, `/issue.*[^\d]*(\d+)/mi`, and `/issue.*[^\d]?(\d+)/mi`, and all expressions catch the exact same set of issue ids. Anyway, I will change the expression to `/issue.*[^\d]*(\d+)/mi` as the current one sounds to restrict.